src/libraries/System.Private.Uri/src/System/Uri.cs

```diff
@@ -5042,105 +5034,73 @@ private static string CombineUri(Uri basePart, string relativePart, UriFormat ur
 
             bool convBackSlashes = basePart.Syntax.InFact(UriSyntaxFlags.ConvertPathSlashes);
 
-            string? left = null;
+            int jointLength = basePart._string.Length + relativePart.Length;
 
             // check for network or local absolute path
             if (c1 == '/' || (c1 == '\\' && convBackSlashes))
             {
                 if (relativePart.Length >= 2 && relativePart[1] == '/')
                 {
                     // got an authority in relative path and the base scheme is not file (checked)
-                    return basePart.Scheme + ':' + relativePart;
+                    return string.Concat(basePart.Scheme, ":", relativePart);
```

Comment:
What's the benefit of the explicit string.Concat call here? I don't mind the change, but this line change isn't actually improving anything; the compiler will already change the ':' to ":" and use the same string.Concat overload as part of the +.

Reply:
Just to be explicit and to match other Concat calls above